src/logging.hpp

```diff
@@ -0,0 +1,63 @@
+#pragma once
+
+#include <google/protobuf/stubs/common.h>
+
+#if GOOGLE_PROTOBUF_VERSION < 3021012
```

Comment:
Let's try to restructure code to check this condition only once and without introducing USE_ABSL_LOGGING.

Reply:
See [ccb30e1](https://github.com/nginxinc/nginx-otel/pull/34/commits/ccb30e11659828c2fb9efa0229cecee0cdcb4237).